integrations/unstructured/pyproject.toml

```
@@ -23,7 +23,12 @@ classifiers = [
   "Programming Language :: Python :: Implementation :: CPython",
   "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = ["haystack-ai", "unstructured", "unstructured-client<0.30.0"]
+dependencies = [
+  "haystack-ai>=2.0.0",
+  "unstructured>=0.15.0",
+  "unstructured-client>=0.21.0"
```

```suggestion
  "unstructured-client>=0.21.0, <0.30.0"
```

If I'm not wrong, we are still incompatible with newer versions of the client and we want to prevent users from installing them

---

no I see that the tests pass, so for some reason, it seems that it works

SORRY